drivers/SmartThings/zigbee-button/src/thirdreality/init.lua

```diff
@@ -0,0 +1,58 @@
+-- Copyright 2022 SmartThings
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+local capabilities = require "st.capabilities"
+
+local function present_value_attr_handler(driver, device, value, zb_rx)
+  local event
+  local additional_fields = {
+    state_change = true
+  }
+  if value.value == 0x0001 then
+    event = capabilities.button.button.pushed(additional_fields)
+    device:emit_event(event)
+  end
+  if value.value == 0x0002 then
+    event = capabilities.button.button.double(additional_fields)
+    device:emit_event(event)
+  end
+  if value.value == 0x0000 then
+    event = capabilities.button.button.held(additional_fields)
+    device:emit_event(event)
+  end
+end
+
+local function device_added(driver, device)
+  device:emit_event(capabilities.button.supportedButtonValues({ "pushed", "double", "held" }))
+  device:emit_event(capabilities.button.numberOfButtons({value = 1}))
+end
+
+local thirdreality_device_handler = {
+  NAME = "ThirdReality Smart Button",
+  lifecycle_handlers = {
+    added = device_added
+  },
+  zigbee_handlers = {
+    attr = {
+      [0x0012] = {
```

Comment:
nit: to improve readability, you could make these values constants at the top of the file. For example 
```
local MULTISTATE_INPUT_ATTR = 0x0012
local PRESENT_VALUE = 0x0055

-----rest of file------

attr = {
  [MULTISTATE_INPUT_ATTR] = {
    [PRESENT_VALUE] = present_value_attr_handler
  }
```